.murdock

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-: ${TEST_BOARDS_AVAILABLE:="nrf52dk samr21-xpro"}
+: ${TEST_BOARDS_AVAILABLE:="esp32-wroom-32 nrf52dk samr21-xpro"}
```

Comment:
This is the only remaining change to enable ESP32 as a test board. With the fixes tracked in issue #12763, all automatic tests are working. The last open PR is #12794. Once tPR #12794 is merged, all tests should work.